src/pudl/analysis/record_linkage/classify_plants_ferc1.py

```diff
@@ -50,24 +50,28 @@
                 embed_dataframe.ColumnCleaner(cleaning_function="null_to_empty_str"),
                 embed_dataframe.CategoricalVectorizer(),
             ],
+            weight=1.0,
```

Comment:
In debugging this issue I played around with these weights, and found having them explicit more legible.

I was surprised that even substantial changes didn't seem to affect the consistency check much at all.  Doubling the weight of `capacity_mw` and `utility_id_ferc1` only increased it from 74.8% to 74.9%.

Reply:
Ya that does seem weird. Maybe that's an indication that inconsistent matches are more attributable to mistakes in the FERC to EIA matching process?